projects/isy-angular-widgets/src/lib/input-char/directives/input-char.directive.spec.ts

```diff
@@ -72,6 +72,16 @@ describe('Integration Tests: InputCharDirective', () => {
     expect(inputCharButton).toBeTruthy();
   });
 
+  it('should set outlinedInputCharButton property correctly', () => {
```

Comment:
Ähnlich wie beim Hauptfenster --> Besser Prüfen ob der Button korrekt gestylt wird nachdem das Prop gesetzt wurde

Reply:
3 neue Tests in input-char.component.spec.ts:
1. should not have outlined style by default for the input char button
2. should have outlined style when outlinedInputCharButton is true
3. should not have outlined style when outlinedInputCharButton is false